study.md

```diff
@@ -27,17 +27,19 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+The DOM is not the html you see when you "view source" on a web page.
+
+Similarly, the DOM is not the html you have written in your file.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes. The information in Dev Tools is a representation of the DOM. This is because the browser fills out any html errors it can fix, and any javascript that changes the DOM will have run. The Dev Tools let you view the result of these changes, aka the DOM.
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.